pyiron_atomistics/sphinx/base.py

```diff
@@ -2009,10 +2017,24 @@ def collect_sphinx_log(self, file_name="sphinx.log", cwd=None):
             self._job.status.aborted = True
         for key, value in results["generic"].items():
             if key not in self.generic:
-                self.generic[key] = value
+                if key == "forces":
+                    self.generic[key] = value * HARTREE_OVER_BOHR_TO_EV_OVER_ANGSTROM
+                elif key == "volume":
+                    self.generic[key] = value * BOHR_TO_ANGSTROM**3
+                else:
+                    self.generic[key] = value
         for key, value in results["dft"].items():
             if key not in self.generic.dft:
-                self.generic.dft[key] = value
+                if key not in [
+                    "scf_energy_int",
+                    "scf_energy_free",
+                    "scf_magnetic_forces",
+                ]:
```

Comment:
```suggestion
                ] and key not in self.generic.dft:
```

Reply:
@samwaseda Thanks for the feedback, I combined the if statements in the commit below.